src/components/PeopleTable/PersonInfo.tsx

```diff
@@ -0,0 +1,43 @@
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person,
+  personMother: Person | undefined,
+  personFather: Person | undefined,
+};
+
+export const PersonInfo: React.FC<Props> = ({
+  person,
+  personMother,
+  personFather,
+}) => {
+  return (
+    <>
+      <td>
+        <PersonLink person={person} />
+      </td>
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
```

Comment:
Consider destructuring the 'person' object to make the code more readable. Example: `const { sex, born, died, motherName, fatherName } = person;`.